RELEASE.md

```diff
@@ -41,8 +41,7 @@
 
 ## Deprecations
 
-
-*   KFP v1 SDK 
+*   KFP v1 SDK
 *   TensorFlow Estimator is Deprecated
```

Comment:
To provide further context, let's include:

Since Estimators will no longer be available in TensorFlow 2.16 and later versions, we have deprecated examples and templates that use them. We encourage you to explore Keras as a more modern and flexible high-level API for building and training models in TensorFlow.